counsel.el

```diff
@@ -4918,6 +4918,36 @@ An extra action allows to switch to the process buffer."
 ;; `counsel-slime-repl-history' within
 ;; `counsel--browse-history-action'.
 
+;;** `counsel-esh-dir-history'
+(declare-function eshell/cd "eshell")
+
+(defun counsel--esh-dir-history-action-cd (pair)
+  "Default action for counsel-esh-dir-history. It changes the
+current working directory to the one selected by the user."
```

Comment:
By convention, the first line of a docstring should comprise a summary of one or two complete sentences.  Subsequent sentences should start on the next line.  See [`(info "(elisp) Documentation Tips")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).

It is also preferable to quote symbols in docstrings with backtick+apostrophe, so that they get hyperlinked, i.e.:
```el
"Default action for `counsel-esh-dir-history'."
```

The same apply to the docstring below.

Reply:
Yup, I'm sorry, I shouldn't have pushed those docstrings. It should be better now.

Reply:
No need to apologise. They're looking much better now, thanks.